### DISCUSSION TITLE
Assistance

### DISCUSSION THREAD

**Discussion by ebuberoderick** [tailwindlabs/tailwindcss#10521] [Ideas]
i need assistance on how to contribute to tailwindcss

**Comment by wongjn** [comment]
The general steps are:
1. Fork the repository.
2. Clone the repository locally.
3. Install the project dependencies.
4. Write a test for the feature/bugfix you are implementing.
5. Make sure all the tests pass by modifying the code on a new branch.
6. Make a PR.

Consider reading the [contribution documentation](https://github.com/tailwindlabs/tailwindcss/blob/master/.github/CONTRIBUTING.md) and perhaps also [Contributing to projects](https://docs.github.com/en/get-started/quickstart/contributing-to-projects).
